docs/rackhd/api-usage-examples.rst

```diff
@@ -45,7 +50,7 @@ This example provides instructions on how to flash a BMC image on a Quanta (node
    The above command will return a SKU ID which will be used to post the Workflow against the node. If an error like "Duplicate entry
    exists" is returned in place of the the SKU ID, check the database and delete a prexisting SKU package.
```

Comment:
PUT method could replace the original SKU, but since the document uses "register" instead of more technical term "POST" or "PUT", create a new sku pack anyway better matches "register". Also, create(POST) and delete seems  to be a more basic operation than update(PUT), so as the basic document, POST and delete might be better than PUT